packages/collapse/src/index.js

```diff
@@ -6,6 +6,7 @@ export default function (Alpine) {
         if (! el._x_isShown) el.style.height = `${floor}px`
         if (! el._x_isShown) el.style.removeProperty('display')
         if (! el._x_isShown) el.style.overflow = 'hidden'
```

Comment:
I think I'll add a `display: none` rule here too, for the reasons given in the PR comments and here: https://github.com/alpinejs/alpine/pull/2353#issuecomment-965709781